tss-esapi/src/context/tpm_commands/non_volatile_storage.rs

```diff
@@ -39,8 +39,8 @@ impl Context {
                 self.optional_session_1(),
                 self.optional_session_2(),
                 self.optional_session_3(),
-                &auth.cloned().unwrap_or_default().into(),
```

Comment:
Interesting that this function was called `cloned` with **d** :thinking:

Reply:
I think it is to distinguish from cloning the reference or cloning the referenced data.

Reply:
Understood, thanks for context. Since now this has been removed maybe that function is no longer needed? Just asking as I never used it :)

Reply:
`cloned`... but after a careful study of other PRs I see it in a few places here and there... I wonder why haven't I seen it earlier :thinking: It's probably all OK now :+1: